<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hash Calculator v 1.1 - september 2019 - By Olivier Michel (olivier.michel@computingnews.net)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="fileLineEdit">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>30</y>
      <width>721</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="filePushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a file to hash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hashPushButton">
    <property name="geometry">
     <rect>
      <x>965</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute hash</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hashResultLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rmd5RadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>246</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha1RadioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>271</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha224RadioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>296</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-224</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha256RadioButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-256</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha512RadioButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>369</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-512</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>190</y>
      <width>1071</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777212</width>
      <height>16777212</height>
     </size>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>1099</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File to hash</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hashed value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verifyLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>1201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify (String will be converted to upper case)</string>
    </property>
   </widget>
   <widget class="QLabel" name="verifyLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify status</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha384RadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>344</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-384</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comparePushButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare hash</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha3_224RadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA3-224</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SHA3_256RadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>322</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA3-256</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SHA_384RadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>346</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA3-384</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SHA3_512RadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA3-512</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
